tests/test_reset.py

```diff
@@ -0,0 +1,154 @@
+"""
+This file was adapted from a setup file given in 
+the https://github.com/microsoft/AirSim-Drone-Racing-Lab repositiry
+It simply loads a level and starts a race against a baseline drone
+to validate that everything has been set up properly.
+"""
+
+import airsimdroneracinglab
+import threading
+import time
+
+
+class ReproduceResetRaceCondition:
+    """
+    Contains vaious functions for interacting with the ADRL unreal environment
```

Comment:
Class docstring needs some reformatting;
 - Needs a section for class attributes
 - Needs a section for class methods: here you write the function definition and a short description of the function

Reply:
Something along the lines of this:
```suggestion
    Contains various functions for interacting with the ADRL unreal environment
    
    Attributes
    -----------
    drone_name: str
        name of the drone
    is_thread_active: bool
        bool to see if current thread is active
    ....
    <repeat; add line for each class variable>
    
    Methods
    ----------
    repeat_timer(callback, period: float)
        basic sleep timer
    load_level(level_name: str, sleep_sec: float)
        Loads a sim level based on the passed name
     ....
    <repeat; add line for each class method>
    
```

Reply:
NGL straight up forgot about this. Sorry for wasting your time, should be completed now.